index.d.ts

```diff
@@ -128,6 +133,14 @@ export const Either: TypeRep;
 export function Left<A, B>(value: A): Either<A, B>;
 export function Right<A, B>(value: B): Either<A, B>;
 
+//  Placeholder
+
+interface Placeholder {
+  '@@functional/placeholder': boolean
```

Comment:
`true` is a literal type.

literal type: `boolean` (`true`, `false`), `number`(`1`, `2`, etc.) and `string` (`'str1'`, `'str2'`, etc.)